manifest.json

```diff
@@ -103,9 +103,9 @@
         }
       },
       "angularAudio": {
-        "version": "1.7.4",
+        "version": "f403ea9e86de141278d410bade8ef1f43e6d188c",
         "downloadFormat": "zip",
-        "url": "https://github.com/danielstern/ngAudio/archive/1.7.4.zip",
+        "url": "https://github.com/danielstern/ngAudio/archive/f403ea9e86de141278d410bade8ef1f43e6d188c.zip",
         "rootDirPrefix": "ngAudio-",
```

Comment:
I think you need to change some of these other fields as well. See e.g. https://github.com/oppia/oppia/blob/01a1c9add12fa083da372d61f9f14f51ee4241ac/manifest.json#L280

Reply:
When rootDir**Prefix** is used the version number gets included by the instalation script. See https://github.com/oppia/oppia/blob/626c4fd98fc44f788d7c7642eb51b8e1362c86dc/scripts/install_third_party.py#L305

Reply:
Yup, but I think we want our directory names to only have a short suffix (so, ngAudio-xxxxxx rather than ngAudio-f403ea9e86de141278d410bade8ef1f43e6d188c) -- hence the reference to the guppy example.

Reply:
ok changed that so the output now gives ngAudio-f403ea as folder name.